lib/src/main/java/org/team100/lib/motion/drivetrain/module/WCPSwerveModule100.java

```diff
@@ -108,20 +110,21 @@ private static PositionServoInterface<Angle100> turningServo(
             int turningEncoderChannel,
             double turningOffset,
             double gearRatio,
-            SwerveKinodynamics kinodynamics) {
+            SwerveKinodynamics kinodynamics,
+            Drive drive) {
         final double turningGearRatio = 1.0;
         FalconTurningMotor turningMotor = new FalconTurningMotor(
                 name,
                 turningMotorCanId,
-                true,
+                false,
```

Comment:
this should be configurable; if you just say "false" here then it breaks the other robot